engine/src/main/java/org/terasology/rendering/dag/nodes/SimpleBlendMaterialsNode.java

```diff
@@ -74,23 +76,11 @@ public void process() {
      */
     private void preRenderSetupSimpleBlendMaterials() {
         READ_ONLY_GBUFFER.setRenderBufferMask(true, true, true);
-        GL11.glEnable(GL_BLEND);
         GL11.glBlendFunc(GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA); // (*)
```

Comment:
By the way, this line should become a StateChange. If you'd like to leave it to after the end of the project, please add a TODO.